---
layout: page
title: Data
---

* TOC
{:toc}

## Phylogenomics

### ASTRAL related

#### weighted ASTRAL:
	
Zhang, C., & Mirarab, S. (2022). Weighting by Gene Tree Uncertainty Improves Accuracy of Quartet-based Species Trees. Molecular Biology and Evolution, 39(12), msac215. [10.1093/molbev/msac215](https://doi.org/10.1093/molbev/msac215).

- Small files, including scripts: [Gihub](https://github.com/chaoszhang/Weighted-ASTRAL_data) 
- Larger files: [Dryad](https://doi.org/10.6076/D1WK5R)

#### ASTRAL-Pro:

- Zhang, C., Scornavacca, C., Molloy, E. K., & Mirarab, S. (2020). ASTRAL-Pro: Quartet-Based Species-Tree Inference despite Paralogy. Molecular Biology and Evolution, 37(11), 3292–3307. [10.1093/molbev/msaa139](https://doi.org/10.1093/molbev/msaa139) 
	- Small files, including scripts: [Gihub](https://github.com/chaoszhang/A-pro_data/) 
	- Larger files: [Dryad](​​https://datadryad.org/stash/dataset/doi:10.6076/D11C7P)

- Zhang, C., & Mirarab, S. (2022). ASTRAL-Pro 2: Ultrafast species tree reconstruction from multi-copy gene family trees. Bioinformatics. [10.1093/bioinformatics/btac620](https://doi.org/10.1093/bioinformatics/btac620)
	- Available on [Gihub](https://github.com/chaoszhang/A-Pro2_data) 



#### ASTRAL-MP:
Yin, J., Zhang, C., & Mirarab, S. (2019). ASTRAL-MP: Scaling ASTRAL to very large datasets using randomization and parallelization. Bioinformatics, 35(20). [doi.org/10.1093/bioinformatics/btz211](https://doi.org/10.1093/bioinformatics/btz211).

* `SV 10-1000`: These gene trees can be found in the ASTRAL-II subsection below.  

* `Avian`: This is the dataset with 48 species and 14446 gene trees with varying contractions. The gene trees, the  species trees generated by ASTRAL as well as the ASTRAL output can be found on [Dryad](https://datadryad.org/stash/dataset/doi:10.6076/D16W2H)

* `Insects`: This is the dataset with 144 species and 149278 gene trees. The gene trees can be found on the [dryad of a different project](https://doi.org/10.6076/D14599) 

* `Simulated 10K`: This is the dataset with 10K species and simulated using SimPhy (20 replicates). 

	* All the data can be found on [Dryad](https://datadryad.org/stash/dataset/doi:10.6076/D16W2H)

	* Scripts to generate the data and to analyze it are also found on [GitHub](https://github.com/smirarab/astralmp-simulations) 

#### ASTRAL-Multiind:
Rabiee, Maryam, Erfan Sayyari, and Siavash Mirarab. “Multi-Allele Species Reconstruction Using ASTRAL.” Molecular Phylogenetics and Evolution 130 (2019): 286–96. [10.1016/j.ympev.2018.10.033](https://doi.org/10.1016/j.ympev.2018.10.033). 

* The smaller data, including script on [GitHub](https://gitlab.com/mrabiee/ASTRAL-multiind/)
* The larger datasets are on [Dryad](https://datadryad.org/stash/dataset/doi:10.6076/D17W2T)

#### INSTRAL:
Rabiee, Maryam, and Siavash Mirarab. “INSTRAL: Discordance-Aware Phylogenetic Placement Using Quartet Scores.” Systematic Biology, July 10, 2019. [doi: 10.1093/sysbio/syz045](https://doi.org/10.1093/sysbio/syz045).

* Data are available on [Dryad](https://doi.org/10.5061/dryad.cs59t13)

#### Polytomy test:
Sayyari, E., & Mirarab, S. (2018). Testing for Polytomies in Phylogenetic Species Trees Using Quartet Frequencies. Genes, 9(3), 132. [doi: 10.3390/genes9030132](https://doi.org/10.3390/genes9030132). 

* Data available on [Github](https://github.com/esayyari/polytomytest), which itself refers to [GitLab](https://gitlab.com/esayyari/polytomy)  

#### ASTRAL-III:
Zhang, Chao, Maryam Rabiee, Erfan Sayyari, and Siavash Mirarab. “ASTRAL-III: Polynomial Time Species Tree Reconstruction from Partially Resolved Gene Trees.” BMC Bioinformatics 19, no. S6 (2018): 153. <https://doi.org/10.1186/s12859-018-2129-y>.

* Data are available on [GitLab](https://gitlab.com/esayyari/ASTRALIII) 

#### LocalPP:
Sayyari, E., & Mirarab, S. (2016). Fast Coalescent-Based Computation of Local Branch Support from Quartet Frequencies. Molecular Biology and Evolution, 33(7), 1654–1668. <https://doi.org/10.1093/molbev/msw079> 

* <https://esayyari.github.io/FastLocalBranchSupport.html>

* **Note:** This is to be moved to a more permanent place soon. 

#### ASTRAL-HGT study:

Davidson, Ruth, Pranjal Vachaspati, Siavash Mirarab, and Tandy Warnow. “Phylogenomic Species Tree Estimation in the Presence of Incomplete Lineage Sorting and Horizontal Gene Transfer.” BMC Genomics 16, no. Suppl 10 (2015): S1. [10.1186/1471-2164-16-S10-S1](https://doi.org/10.1186/1471-2164-16-S10-S1).

* Data available from [Illinois data bank](https://databank.illinois.edu/datasets/IDB-6670066).

#### ASTRAL-II:
Mirarab, S., & Warnow, T. (2015). ASTRAL-II: Coalescent-based species tree estimation with many hundreds of taxa and thousands of genes. Bioinformatics, 31(12), i44–i52. [10.1093/bioinformatics/btv234](https://doi.org/10.1093/bioinformatics/btv234) 

* Simulation scripts are available on [Github](https://github.com/smirarab/astral2sims)

* All the simulation and real data files are available on [Dryad](https://doi.org/doi:10.6076/D10C7C).

#### ASTRAL-I: 
Mirarab, Siavash, Rezwana Reaz, Md. Shamsuzzoha Bayzid, Théo Zimmermann, M. S. Swenson, and Tandy Warnow. “ASTRAL: Genome-Scale Coalescent-Based Species Tree Estimation.” Bioinformatics 30, no. 17 (2014): i541–48. [doi:10.1093/bioinformatics/btu462](doi.org/10.1093/bioinformatics/btu462).

* Data will be **on Dryad** soon.


### Binning related

#### Statistical Binning

Mirarab, Siavash, Md. Shamsuzzoha Bayzid, Bastien Boussau, and Tandy Warnow. “Statistical Binning Enables an Accurate Coalescent-Based Estimation of the Avian Tree.” Science 346, no. 6215 (2014): 1250463–1250463. [doi: 10.1126/science.1250463](http://dx.doi.org/10.1126/science.1250463)  

* The statistical binning dataset is permanently provided at UIUC under [doi:10.13012/C5MW2F2P](http://dx.doi.org/10.13012/C5MW2F2P) 
* Some files are missing from that link and are provided on [this GitHub page](https://github.com/smirarab/binning-extra-data). The README.pdf file shown on that github page gives the details. 

#### Weighted Statistical Binning


Bayzid, Md. Shamsuzzoha, Siavash Mirarab, Bastien Boussau, and Tandy Warnow. “Weighted Statistical Binning: Enabling Statistically Consistent Genome-Scale Phylogenetic Analyses.” PLoS ONE 10, no. 6 (January 18, 2015): e0129183. doi:10.1371/journal.pone.0129183.

* Most of the datasets used in this study are available through the prior publication (statistical binning). 

* The new datasets generated for this study are available on [figshare, with DOI: 10.6084/m9.figshare.1411146](http://dx.doi.org/10.6084/m9.figshare.1411146). 
	* A more detailed RADME files is available [here](https://github.com/smirarab/binning-extra-data/tree/main/weighted-binning) 

* The weighted statistical binning software is available on [GitHub](https://github.com/smirarab/binning).

#### Response to comment on statistical binning

Mirarab, Siavash, Md. Shamsuzzoha Bayzid, Bastien Boussau, and Tandy Warnow. “Response to Comment on ‘Statistical Binning Enables an Accurate Coalescent-Based Estimation of the Avian Tree.’” Science 350, no. 6257 (2015): 171. [doi:10.1126/science.aaa7719](https://doi.org/10.1126/science.aaa7719).

* Data are made available on [GitHub](https://github.com/smirarab/binning-extra-data); See `binning-response` directory there. 

#### BBCA
Zimmermann, Théo, Siavash Mirarab, and Tandy Warnow. “BBCA: Improving the Scalability of *BEAST Using Random Binning.” BMC Genomics 15, no. Suppl 6 (October 17, 2014): S11. [doi:10.1186/1471-2164-15-S6-S11](https://doi.org/10.1186/1471-2164-15-S6-S11).

* Data are made available on [GitHub](https://github.com/smirarab/binning-extra-data/tree/main/bbca); see `bbca` directory there. 

### Misc

#### uDance

Metin Balaban et al., “Generation of Accurate, Expandable Phylogenomic Trees with uDance,” Nature Biotechnology Online (2023), <https://doi.org/10.1038/s41587-023-01868-8>.

* Data is available from Harvard repository <https://doi.org/10.7910/DVN/BCUM6P>
* See also <https://github.com/balabanmetin/uDANCE-data>

#### Quco
Maryam Rabiee and Siavash Mirarab, “QuCo: Quartet-Based Co-Estimation of Species Trees and Gene Trees,” Bioinformatics 38, no. Supplement_1 (2022): i413–21, <https://doi.org/10.1093/bioinformatics/btac265>.

* See <https://gitlab.com/mrabiee/quo-data>
* Most of the data are on Dryad: <https://datadryad.org/stash/dataset/doi:10.6076/D1CP4R>

#### TripVote

Mai, Uyen, and Siavash Mirarab. “Completing Gene Trees without Species Trees in Sub-Quadratic Time.” Edited by Russell Schwartz. Bioinformatics 38, no. 6 (M2022): 1532–41. <https://doi.org/10.1093/bioinformatics/btab875>.


* See <https://uym2.github.io/tripVote/>
* The actual data are on [Dryad](https://datadryad.org/stash/dataset/doi:10.6076/D1N30V)


#### 10K Bacteria:
Zhu, Q., Mai, U., Pfeiffer, W., Janssen, S., Asnicar, F., Sanders, J. G., Belda-Ferre, P., Al-Ghalith, G. A., Kopylova, E., McDonald, D., Kosciolek, T., Yin, J. B., Huang, S., Salam, N., Jiao, J., Wu, Z., Xu, Z. Z., Cantrell, K., Yang, Y., … Knight, R. (2019). Phylogenomics of 10,575 genomes reveals evolutionary proximity between domains Bacteria and Archaea. Nature Communications, 10(1), 5477. [doi: 10.1038/s41467-019-13443-4](https://doi.org/10.1038/s41467-019-13443-4) 

* The 10k bacteria dataset is available [here](https://github.com/biocore/wol)

#### Fragmentary data:
Erfan Sayyari, James B Whitfield, and Siavash Mirarab, “Fragmentary Gene Sequences Negatively Impact Gene Tree and Species Tree Reconstruction,” Molecular Biology and Evolution 34, no. 12 (2017): 3279–91, <https://doi.org/10.1093/molbev/msx261>.

* See [this page](http://esayyari.github.io/fragments.html) 
* Most of the data is on [Dryad: doi:10.6076/D14599](https://datadryad.org/stash/dataset/doi:10.6076/D14599)


#### DISTIQUE
Sayyari, Erfan, and Siavash Mirarab. “Anchoring Quartet-Based Phylogenetic Distances and Applications to Species Tree Reconstruction.” BMC Genomics 17, no. S10 (2016): 101–13. [doi: 10.1186/s12864-016-3098-z](https://doi.org/10.1186/s12864-016-3098-z).

* http://esayyari.github.io/DISTIQUE 
* Some of the files linked there may be lost, unfortunately. 


## Skimming

#### APPLES-I: 

* Available on [Dryad](https://datadryad.org/stash/dataset/doi:10.5061/dryad.78nf7dq)

#### APPLES-II: 

* Available on [Zenodo](https://zenodo.org/record/5551285)

#### MISA: 
Balaban, Metin, and Siavash Mirarab. “Phylogenetic Double Placement of Mixed Samples.” Bioinformatics 36, no. Supplement_1 (July 1, 2020): i335–43. [doi: 10.1093/bioinformatics/btaa489](https://doi.org/10.1093/bioinformatics/btaa489).


* Smaller files on [GitHub](https://github.com/balabanmetin/misa-data)
* Large files are given on [Dryad](https://datadryad.org/stash/dataset/doi:10.6076/D1QW25)

#### Skmer

* https://shahab-sarmashghi.github.io/Skmer/
* https://skmer.ucsd.edu/data/skmer


## Microbiome

## Branch Length

### Dating

#### wLogDate

Uyen Mai, Siavash Mirarab, Log Transformation Improves Dating of Phylogenies, Molecular Biology and Evolution, msaa222, https://doi.org/10.1093/molbev/msaa222.

* https://uym2.github.io/wLogDate/ 

### Rooting, Clustering, Error detection, etc.
#### MinVarRooting

Mai, Uyen, Erfan Sayyari, and Siavash Mirarab. “Minimum Variance Rooting of Phylogenetic Trees and Implications for Species Tree Reconstruction.” Edited by Gabriel Moreno-Hagelsieb. PLOS ONE 12, no. 8 (August 11, 2017): e0182238. https://doi.org/10.1371/journal.pone.0182238.

* For data, see https://uym2.github.io/MinVar-Rooting/ 

##### TreeShrink

Mai, Uyen, and Siavash Mirarab. “TreeShrink: Fast and Accurate Detection of Outlier Long Branches in Collections of Phylogenetic Trees.” BMC Genomics 19, no. S5 (May 8, 2018): 272. [doi: 10.1186/s12864-018-4620-2](https://doi.org/10.1186/s12864-018-4620-2). 

* For dasta, see [https://uym2.github.io/TreeShrink/](https://uym2.github.io/TreeShrink/)
* Much of the larger datasets are on [Dryad](https://doi.org/doi:10.6076/D1HC71)

#### TreeCluster
Balaban, Metin, Niema Moshiri, Uyen Mai, Xingfan Jia, and Siavash Mirarab. “TreeCluster: Clustering Biological Sequences Using Phylogenetic Trees.” Edited by Serdar Bozdag. PLOS ONE 14, no. 8 (2019): e0221068. [doi:10.1371/journal.pone.0221068](https://doi.org/10.1371/journal.pone.0221068).

* Data deposited on [Zenodo](https://zenodo.org/record/3358386)


## Cladogenesis 

#### DIMSIM

* https://datadryad.org/stash/dataset/doi:10.6076/D1530H 

#### DualBirth model

Moshiri and Mirarab, Dual-birth: A Two-State Model of Tree Evolution and Its Applications to Alu Retrotransposition. 10.1093/sysbio/syx088

* https://doi.org/10.5061/dryad.13n52 


## Obsolete

This [dataset webpage](https://sites.google.com/eng.ucsd.edu/datasets/home?authuser=0) used to host our datasets. We have made an effort to make everything available elsewhere. I provide the link here just in case it becomes useful. 
