---
layout: page
title: Datasets
---

# Phylogenomics

## ASTRAL related

#### weighted ASTRAL:
	
Zhang, C., & Mirarab, S. (2022). Weighting by Gene Tree Uncertainty Improves Accuracy of Quartet-based Species Trees. Molecular Biology and Evolution, 39(12), msac215. https://doi.org/10.1093/molbev/msac215.

- Small files, including scripts: [Gihub](https://github.com/chaoszhang/Weighted-ASTRAL_data) 
- Larger files: [Dryad](https://doi.org/10.6076/D1WK5R)

#### ASTRAL-Pro:

- Zhang, C., Scornavacca, C., Molloy, E. K., & Mirarab, S. (2020). ASTRAL-Pro: Quartet-Based Species-Tree Inference despite Paralogy. Molecular Biology and Evolution, 37(11), 3292–3307. https://doi.org/10.1093/molbev/msaa139 
	- Small files, including scripts: [Gihub](https://github.com/chaoszhang/A-pro_data/) 
	- Larger files: [Dryad](​​https://datadryad.org/stash/dataset/doi:10.6076/D11C7P)

- Zhang, C., & Mirarab, S. (2022). ASTRAL-Pro 2: Ultrafast species tree reconstruction from multi-copy gene family trees. Bioinformatics. https://doi.org/10.1093/bioinformatics/btac620 
	- Small files, including scripts: [Gihub](https://github.com/chaoszhang/A-Pro2_data) 
	- Larger files: [Dryad]()



#### ASTRAL-MP:
Yin, J., Zhang, C., & Mirarab, S. (2019). ASTRAL-MP: Scaling ASTRAL to very large datasets using randomization and parallelization. Bioinformatics, 35(20). https://doi.org/10.1093/bioinformatics/btz211.



* `SV 10-1000`: These gene trees can be found in the ASTRAL-II subsection below.  

* `Avian`: This is the dataset with 48 species and 14446 gene trees with varying contractions. The gene trees, the  species trees generated by ASTRAL as well as the ASTRAL output can be found on [Dryad](https://datadryad.org/stash/dataset/doi:10.6076/D16W2H)

* `Insects`: This is the dataset with 144 species and 149278 gene trees. The gene trees can be found on the [dryad of a different project](https://doi.org/10.6076/D14599) 

* `Simulated 10K`: This is the dataset with 10K species and simulated using SimPhy (20 replicates). 

	* All the data can be found on [Dryad](https://datadryad.org/stash/dataset/doi:10.6076/D16W2H)

	* Scripts to generate the data and to analyze it are also found on [GitHub](https://github.com/smirarab/astralmp-simulations) 

#### ASTRAL-Multiind:
Rabiee, Maryam, Erfan Sayyari, and Siavash Mirarab. “Multi-Allele Species Reconstruction Using ASTRAL.” Molecular Phylogenetics and Evolution 130 (2019): 286–96. https://doi.org/10.1016/j.ympev.2018.10.033. 

* The smaller data, including script on [GitHub](https://gitlab.com/mrabiee/ASTRAL-multiind/)
* The larger datasets are on [Dryad](https://datadryad.org/stash/dataset/doi:10.6076/D17W2T)

#### INSTRAL:
Rabiee, Maryam, and Siavash Mirarab. “INSTRAL: Discordance-Aware Phylogenetic Placement Using Quartet Scores.” Systematic Biology, July 10, 2019. https://doi.org/10.1093/sysbio/syz045.

* Data are available on [Dryad](https://doi.org/10.5061/dryad.cs59t13)

#### Polytomy test:
Sayyari, E., & Mirarab, S. (2018). Testing for Polytomies in Phylogenetic Species Trees Using Quartet Frequencies. Genes, 9(3), 132. https://doi.org/10.3390/genes9030132 

* Data available on [Github](https://github.com/esayyari/polytomytest), which itself refers to [GitLab](https://gitlab.com/esayyari/polytomy)  

#### ASTRAL-III:
Zhang, Chao, Maryam Rabiee, Erfan Sayyari, and Siavash Mirarab. “ASTRAL-III: Polynomial Time Species Tree Reconstruction from Partially Resolved Gene Trees.” BMC Bioinformatics 19, no. S6 (2018): 153. https://doi.org/10.1186/s12859-018-2129-y.

* Data are available on [GitLab](https://gitlab.com/esayyari/ASTRALIII) 

#### LocalPP:
Sayyari, E., & Mirarab, S. (2016). Fast Coalescent-Based Computation of Local Branch Support from Quartet Frequencies. Molecular Biology and Evolution, 33(7), 1654–1668. https://doi.org/10.1093/molbev/msw079 

https://esayyari.github.io/FastLocalBranchSupport.html

**Note:** This is to be moved to a more permanent place soon. 

#### ASTRAL-HGT study:
Davidson, Ruth; Vachaspati, Pranjal; Mirarab, Siavash; Warnow, Tandy (2023): Data from: Phylogenomic species tree estimation in the presence of incomplete lineage sorting and horizontal gene transfer. 

* Data available from [Illinois data bank](https://databank.illinois.edu/datasets/IDB-6670066).

#### ASTRAL-II:
Mirarab, S., & Warnow, T. (2015). ASTRAL-II: Coalescent-based species tree estimation with many hundreds of taxa and thousands of genes. Bioinformatics, 31(12), i44–i52. https://doi.org/10.1093/bioinformatics/btv234 

* Simulation scripts are available on [Github](https://github.com/smirarab/astral2sims)

* All the simulation and real data files are available on [Dryad](https://doi.org/doi:10.6076/D10C7C).

#### ASTRAL-I: 
Mirarab, Siavash, Rezwana Reaz, Md. Shamsuzzoha Bayzid, Théo Zimmermann, M. S. Swenson, and Tandy Warnow. “ASTRAL: Genome-Scale Coalescent-Based Species Tree Estimation.” Bioinformatics 30, no. 17 (2014): i541–48. [doi:10.1093/bioinformatics/btu462](doi.org/10.1093/bioinformatics/btu462).

* Data will be **on Dryad** soon.



## Binning related

# Skimming

* APPLES-I: https://datadryad.org/stash/dataset/doi:10.5061/dryad.78nf7dq

* APPLES-II: https://zenodo.org/record/5551285 

# Microbiome

# Branch Length

Mai, Uyen, Erfan Sayyari, and Siavash Mirarab. “Minimum Variance Rooting of Phylogenetic Trees and Implications for Species Tree Reconstruction.” Edited by Gabriel Moreno-Hagelsieb. PLOS ONE 12, no. 8 (August 11, 2017): e0182238. https://doi.org/10.1371/journal.pone.0182238.

For data, see https://uym2.github.io/MinVar-Rooting/ 

Mai, Uyen, and Siavash Mirarab. “TreeShrink: Fast and Accurate Detection of Outlier Long Branches in Collections of Phylogenetic Trees.” BMC Genomics 19, no. S5 (May 8, 2018): 272. https://doi.org/10.1186/s12864-018-4620-2. 

For dasta, see https://uym2.github.io/TreeShrink/ 

Balaban, Metin, Niema Moshiri, Uyen Mai, Xingfan Jia, and Siavash Mirarab. “TreeCluster: Clustering Biological Sequences Using Phylogenetic Trees.” Edited by Serdar Bozdag. PLOS ONE 14, no. 8 (August 22, 2019): e0221068. https://doi.org/10.1371/journal.pone.0221068.

https://zenodo.org/record/3358386

Uyen Mai, Siavash Mirarab, Log Transformation Improves Dating of Phylogenies, Molecular Biology and Evolution, msaa222, https://doi.org/10.1093/molbev/msaa222.

https://uym2.github.io/wLogDate/ 

# Misc

Moshiri and Mirarab, Dual-brith
A Two-State Model of Tree Evolution and Its Applications to Alu Retrotransposition
10.1093/sysbio/syx088
https://doi.org/10.5061/dryad.13n52 


Sayyari and Mirarab,
Fragmentary data
Fragmentary Gene Sequences Negatively Impact Gene Tree and Species Tree Reconstruction
10.1093/molbev/msx261
http://esayyari.github.io/fragments.html 

Sayyari and Mirarab, DISTIQUE
Anchoring quartet-based phylogenetic distances and applications to species tree reconstruction
10.1186/s12864-016-3098-z
http://esayyari.github.io/DISTIQUE 



