---
layout: page
title: Tools/Data
---

## Software

* See my [gitub page](https://github.com/smirarab) to see all my publicly available software programs.
  Many of these are from my days as a graduate student advised by [Tandy Warnow](http://tandy.cs.illinois.edu) 
    * [ASTRAL](https://github.com/smirarab/ASTRAL): Species tree estimation from unrooted gene trees using a quartet-based approach (consistent under multi-species coalescent process and hence useful for handling gene tree incongruence caused by ILS). 
    * [PASTA](http://www.cs.utexas.edu/~phylo/software/pasta/): Software for alignment and tree estimation on very large datasets (thousands to hundreds of thousands of sequences). This is a new version of SATé, and the code is heavily based on the SATé code. 
    * [SEPP](http://www.cs.utexas.edu/~phylo/software/sepp/submission/): Software for fast phylogenetic placement based on SATe decomposition - useful for metagenomic analyses
    * [TIPP](http://www.cs.utexas.edu/~phylo/software/sepp/tipp-submission): Taxonomic identification and taxonomic profiling of metagenomic samples (uses SEPP). 
    * [UPP](http://www.cs.utexas.edu/~phylo/software/upp/): Ultra-large alignments using Phylogeny-aware Profiles (uses SEPP). 
    * [FastSP](http://www.cs.utexas.edu/~smirarab/fastsp): Fast and Memory efficient software for comparison of multiple sequence alignments (computes SP-FN and TC scores) 
    * [Statistical binning](https://github.com/smirarab/binning): the code to perform statistical binning
    * [Misc](https://github.com/smirarab/global): a patchwork of scripts for phylogenetics and MSA that may be useful to others
    * I also work on and contribute code to [SATé](http://phylo.bio.ku.edu/software/sate/sate.html). 
    
* Also checkout the following github pages from students in the lab.
    * [Uyen Ma](https://github.com/uym2):
        * [Minimum Variance (MV) rooting](https://github.com/uym2/MinVar-Rooting): also includes an efficient implementation of the MP rooting
        * [TreeShrink](https://github.com/uym2/TreeShrink): a tool for detecting extremely and suspiciously long branches in a tree
    * [Erfan](https://github.com/esayyari): 
        * [DISTIQUE](https://esayyari.github.io/DISTIQUE.html)
        * [DiscoVista](https://github.com/esayyari/DiscoVista)
    * [Niema](https://github.com/niemasd/)
        * [FAVITES](https://github.com/niemasd/FAVITES)
        * [Dual-birth](https://github.com/niemasd/Dual-Birth-Model): a set of scripts used in our draft paper that introduced a dual-birth evolutionary process
    

## Datasets

Refer to [this site](https://sites.google.com/eng.ucsd.edu/datasets/home) for most of the datasets we have published through years.     
